3/3scale-istio-adapter/Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+MAINTAINER Krishna Harsha Voora <krishvoor@in.ibm.com>
+
+
+# Install Prerequisites
+RUN yum install -y git make gcc wget
```

Comment:
If you've installed these before the build, you probably want to remove them after the build to keep the container smaller.

Reply:
The "life" of the container is short-spanned, and its purpose being is spinning it up for a quick build and test.
I will add the changes.